app/controllers/backgrounds_controller.rb

```diff
@@ -25,7 +25,18 @@ def edit
   # POST /backgrounds
   # POST /backgrounds.json
   def create
-    @background = Background.new(background_params)
+    @background = Background.create(background_params)
+
+    data = @background.summarize.map { |h| h.slice('tax_id', 'count_type', 'tax_level', 'name', :background_id, :created_at, :updated_at, :mean, :stdev) }
+    columns = data.first.keys
+    values_list = data.map do |hash|
+      hash.values.map do |value|
```

Comment:
Are the values here always in the same order for each of the hashes?

Reply:
yes: https://stackoverflow.com/questions/31418673/is-order-of-a-ruby-hash-literal-guaranteed